server/src/test/java/io/crate/integrationtests/PartitionedTableIntegrationTest.java

```diff
@@ -1635,18 +1635,6 @@ public void testPartitionedTableNestedPk() throws Exception {
         assertThat(printedTable(response.rows())).isEqualTo("t| 04132\n");
     }
 
-
-    @Test
-    public void testCreateTableWithIllegalCustomSchemaCheckedByES() {
```

Comment:
Isn't this test this useful?

Reply:
Right, my fault. Requires to move indexName validation up to the `TransportCreateTable` now, will fix. thx for catching!